Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -2141,7 +2163,7 @@ def mirror_investigation():
         service_url: str = integration_context.get('service_url', '')
         server_links: dict = demisto.demistoUrls()
         server_link: str = server_links.get('server', '')
-        warroom_link = f'{server_link}/#/WarRoom/{investigation_id}'
+        warroom_link = f"{server_link + '/#' if not is_demisto_version_ge('8.0.0') else server_link}/WarRoom/{investigation_id}"
```

Comment:
same here consider setting the server_link with /# in a line before 2166. 
This line as it is now is not very readable